contributing/code/standards.rst

```diff
@@ -167,6 +167,24 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-conditions:
+
+Conditions
+----------
+
+Use `Yoda condition`_ when checking a variable against a value to avoid an
+accidental assignment inside the condition statements::
```

Comment:
Moved inside "structure" section. It would be nice to know what others think about @Soullivaneuh proposal.